lib/render/editor/editor.html.erb

```diff
@@ -872,6 +872,10 @@
         else if (reason === "cancel" && editor.interactiveMode) {
           this._onInteractiveStop(editor);
         }
+        else if (reason === "cancel" && !editor.interactiveMode) {
```

Comment:
The `&&` is not necessary, since it can only be true at that position

Reply:
I've addressed this one on my [last commit](https://github.com/mumuki/mumuki-gobstones-runner/pull/223/commits/7b4260776d85af0a1f35c9bab3e5b0b258682c08)